src/withTranslators.js

```diff
@@ -1,11 +1,13 @@
-import { getContext } from 'recompose'
+import { getContext, wrapDisplayName } from 'recompose'
 
 import * as contextTypes from './contextTypes.js'
 
 /**
  * Provides the given component with translator functions
  * as props.
  */
-export default function withTranslators(Component) {
-  return getContext(contextTypes)(Component)
+export default function withTranslators(WrappedComponent) {
+  const WithTranslators = getContext(contextTypes)(WrappedComponent)
+  WithTranslators.displayName = wrapDisplayName(WrappedComponent, 'WithTranslators')
+  return WithTranslators
```

Comment:
I'm thinking whether it might be more elegant to use `compose()` (part of the recompose library) here, like:

```js
return compose(
  wrapDisplayName('withTranslators'),
  getContext(contextTypes)
)(Component)
```